### ISSUE TITLE
Bond#getQuery() in java wrapper returns object of type SWIGTYPE_p_Queries__QueryT_int_RDKit__Bond_const_p_true_t

### ISSUE THREAD

**Issue by sroughley** [rdkit/rdkit#1279]
The type returned by the getQuery() method of the Bond / QueryBond classes in the Java wrappers in a SWIG object which cannot be utilised other than in the #setQuery() method e.g.

`ROMol query = RWMol.MolFromSmarts("[*]-!@[*]");
SWIGTYPE_p_Queries__QueryT_int_RDKit__Bond_const_p_true_t bondQuery = query.getBondWithIdx(0).getQuery();`

**Comment by sroughley** [comment]
Further investigation shows the same is also true for the corresponding method on the Atom objects.

**Comment by greglandrum** [comment]
This is happening because we haven't wrapped query objects. I'm not sure how difficult that would be, but I'll look into it.

**Comment by sroughley** [comment]
Thanks Greg.  I had assumed that was what was causing the problem.  I had a quick look myself but am not really familiar enough with either the query code of the SWIG wrapper-generating code to get anywhere with it without some serious time

**Comment by greglandrum** [comment]
Yeah, it should be fixable, but there are a couple of non-trivial aspects of it. I'll try to get something done in the next week or so.

**Comment by sroughley** [comment]
The 'non-trivial aspects' possibility was what really worried me!  Thanks - not an urgent but would make a few things simpler if/when it happens

**Comment by sroughley** [comment]
Just came across this in a comment in my code during review - any further update on it @greglandrum ?
